velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -50,9 +56,21 @@ class NthValueTest : public WindowTestBase {
     testWindowFunction(input, "nth_value(c3, c2)");
   }
 
-  // This is for testing different output column types in the nth_value
-  // column parameter.
-  void testPrimitiveType(const TypePtr& type) {
+  void testFirstLastValue(const std::vector<RowVectorPtr>& input) {
```

Comment:
Please add a testValueFunctions() function that calls both testNthValue and testFirstLastValue() and then use that function in all the unit tests.